### ISSUE TITLE
3 features not working: legend, marker.attr & countries.attr

### ISSUE THREAD

**Issue by Ths2-9Y-LqJt6** [Packet-Clearing-House/maptable#3]
@melalj  - I have what I think is correct JS calls to these three objects, but the MapTable fails to render them.

First up is the `contries.attr.r` call that doesn't affect the map, but no JS errors in console:

```javascript
countries: {
    rollup: function(a) {
        return a.length;
    },
    attr: { // todo  this doesn't work
        fill: {
            min: "black",
            max: "white",
            empty: "red",
        },
        stroke: "#d9d9d9",
        "stroke-width": 0.5
    },
    attrEmpty: {
        fill: "#f9f9f9"
    }
```

next up is the `markers.attr` call, which again doesn't affect the map but throws now JS errors:

```javascript
markers: {
    groupBy: function(a) {
        return a.city + ", " + a.country;
    },
    rollup: function(a) {
        return a.length;
    },
    attr: {
        r: { // todo this doesn't work
            min: "minValue",
            max: "maxValue",
            transform: function(v) {
                return 3 * Math.sqrt(v);
            },
        },
        fill: "yellow",
        stroke: "#d9d9d9",
        "stroke-width": 0.5
    },
    tooltip: function(a) {
        out = '<div class="arrow"></div>';
        out += '<span class="badge pull-right"> ' + a.values.length + '</span><h3 class="popover-title"> ' + a.key + '</h3>';
        out += '<div class="popover-content">';
        for (i = 0; i < a.values.length; i++) out += " • " + a.values[i].long_name + "<br>";
        out += "</div>";
        return out;
    },
},
```

finally,  the `legend` boolean:

```javascript
legend: true, // todo this doesn't work
```

**Comment by melalj** [comment]
Fixed on the last push

I moved the legend to the attributes (at the moment it works only for countries.attr.fill). 
Check more details on the README

**Comment by Ths2-9Y-LqJt6** [comment]
@melalj  - there's no mention of `legend` in the docs.  Well, there is, but not for documenting it's use ;)  We mention it in the examples section at the top and and the ScaledValue section, but no explicit mention on how to use it.

`contries.attr.r` is still not working for the colors of the country and legend (assuming new example has legend usage correct). I may have the syntax wrong?

`markers.attr` isn't working for me either.

**Comment by melalj** [comment]
What is the code sample that you used?

Check out the example list.

**Comment by Ths2-9Y-LqJt6** [comment]
@melalj  - ahah!  I was using FF latest.  Following your advice, I was able to reproduce all three problems using the [05 IXPDIR sample](https://github.com/Packet-Clearing-House/maptable/tree/refactor/examples/05%20-%20IXP%20Dir) code.  

tl;dr - works in chromium & chrome, safari, fails in FF. 

We'll work on IE after this, as it fails in a different way on IE10 ;)

**Comment by melalj** [comment]
Fixed on https://github.com/Packet-Clearing-House/maptable/commit/7b92512d79be09af9b8a06a4a1f7a52ad911e225

Still maybe to try on IE10 :)
